CONTRIBUTING.md

```diff
@@ -351,6 +351,226 @@ The third schema is `third.schema.json`, it extends both `second`, and transitiv
 }
 ```
 
+### Schema Descriptors
+
+Schema descriptors are an extensible mechanism for providing additional metadata about an XDM schema. For example, schema descriptors can be used to define relationships between schemas or to annotate schema properties with additional metadata. Schema descriptors may be used when certain properties of a schema are not static (which could usually be described in the schema directly) but may vary from usage to usage.
+
+Details on using and defining schema descriptors may be found in [SCHEMA_DESCRIPTORS.md](./SCHEMA_DESCRIPTORS.md)
```

Comment:
We might want to move that into the `docs` folder (and give it a lowercase name)

Reply:
After reading `SCHEMA_DESCRIPTORS.md` in full, some additional comments:

- definitely move it into `docs` and make it an official part of the XDM documentation
- remove duplicative content from `CONTRIBUTING.md`, just point out that there are schema descriptors, that they can be defined within the schema and outside it, recommend to have schema descriptors for core XDM schemas inside the schema (not standalone)
- For each schema descriptor, create a `description.md` file with examples and usage instructions, so that we can have discussions about the semantics and direction of schemas like the one-to-many descriptor there (because they don't apply to the framework in general, but to the specific descriptor)
- shorten the examples in `SCHEMA_DESCRIPTORS.md` (and remove them entirely from `CONTRIBUTING.md`), as they can be handled better and with more detail in the reference documentation.